src/FreeDSx/Snmp/Protocol/TrapProtocolHandler.php

```diff
@@ -41,22 +41,6 @@ class TrapProtocolHandler
      */
     protected $listener;
 
-    /**
-     * @var array
-     */
-    protected $options = [
-        'timeout_connect' => 5,
-        'timeout_read' => 10,
-        'ssl_validate_cert' => true,
-        'ssl_allow_self_signed' => null,
-        'ssl_ca_cert' => null,
-        'ssl_peer_name' => null,
-        'whitelist' => null,
-        'version' => null,
-        'community' => null,
-        'engine_id' => null,
-    ];
-
```

Comment:
I'd rather these remain initialized here than in the constructor.

Reply:
But then it would be difficult to merge it as is done via `$options + [` here (?)

Reply:
This also makes the property dynamically created in the constructor though? Would like to avoid that. Could still be merged via ` $this->options = $options + $this->options` ? Or am I missing something?

Reply:
Yea, probably could. What about now. I don't fancy having initial value for property that is overwritten at a time of instantiation, constant is fine tho, hm?

Reply:
The constant holding the defaults look fine. Could the `$options` still be initialized in the class though? Just as:

```php
/**
 * @var array
 */
protected $options;
```
With / without the phpdoc is fine. Would prefer to have all the properties initialized in the class, as it makes it obvious as to what is all in use in the class at a glance.

Reply:
This property is in trait IdGenerator so with current state I'd leave it like this. However I'd fancy to get rid of the trait in the future. Then we can set the constant as a default value instead of `[]`.

Reply:
Ah, good point. Didn't notice this addition.